budapp/client/src/signup.js

```diff
@@ -43,7 +43,6 @@ const SignUp = () => {
       if (response.ok) {
         // Successful sign-up logic here
         console.log("User signed up successfully!");
-        window.location.href = "http://localhost:3000/transactions";
```

Comment:
Is there alternative to removing this line? Have to move client to transaction page somehow

Reply:
I believe we can do some react reroute

Reply:
Yeah for this I got an error one time because I wasn't running on port 3000 and instead 3001. There's def a way to use routes with this tho.

Reply:
Ill investigate